README.md

```diff
@@ -104,17 +97,75 @@ docker-compose build -- builder batch_submitter
 docker-compose start batch_submitter
 ```
 
+Source code changes can have an impact on more than one container.
+**If you're unsure about which containers to rebuild, just rebuild them all**:
+
+```
+cd ops
+docker-compose down
+docker-compose build --parallel
+docker-compose up
+```
+
+Finally, **if you're running into weird problems and nothing seems to be working**, run:
+
+```
+cd optimism
+yarn clean
+yarn build
+cd ops
+docker-compose down -v
+docker-compose build --parallel
```

Comment:
I've had issues with `--parallel` when building `builder deployer` - it would build in parallel using the currently built `builder` instead of waiting for `builder` to finish building so that `deployer` can build from it